projects/assessment-summary-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/AssessmentService.kt

```diff
@@ -27,7 +27,7 @@ class AssessmentService(
     private val eventRepository: EventRepository,
     private val contactService: ContactService
 ) {
-    fun recordAssessment(person: Person, summary: AssessmentSummary) {
+    fun recordAssessment(person: Person, summary: AssessmentSummary, telemetryParams: Map<String, String>) {
```

Comment:
Is `telemetryParams` being used anywhere?

Reply:
was going to but decided not to. Removed